fendermint/docker/builder.ci.Dockerfile

```diff
@@ -85,5 +86,5 @@ RUN set -eux; \
   amd64) ARCH='x86_64'  ;; \
   arm64) ARCH='aarch64' ;; \
   esac; \
-  cargo install --locked --root output --path fendermint/app --target ${ARCH}-unknown-linux-gnu &&\
+  cargo install --locked --root output --path fendermint/app --target ${ARCH}-unknown-linux-gnu ; \
```

Comment:
hmm, why did you change this (and above)? I guess there is no possibility of hiding errors I guess but still the previous version _feels_ safer :S

Reply:
I just wanted to see if it makes any difference, although didn't really expect it to do. The `set -eux;` in the beginning causes all this script to fail if anything goes wrong, and because the previous steps were all separated by `;` I thought we should use the same throughout. I don't have a strong opinion, though.